github/AppAuthentication.py

```diff
@@ -49,9 +49,13 @@ def __init__(
         private_key: str,
         installation_id: int,
         token_permissions: Optional[Dict[str, str]] = None,
+        token_repositories: Optional[List[str]] = None,
+        token_repository_ids: Optional[List[int]] = None,
```

Comment:
What about unifying:
```suggestion
        token_repositories: Optional[List[str | int]] = None,
```
just because 🦄.